lib/react_client/js_backed_map.dart

```diff
@@ -38,7 +38,8 @@ class JsBackedMap extends MapBase<dynamic, dynamic> {
 
   // Private helpers with narrower typing than we want to expose, for use in other methods
   List<dynamic> get _keys => _Object.keys(jsObject);
-  List<dynamic> get _values => _Object.values(jsObject);
+  // Use keys to access the value instead oof `Object.values` since MSIE 11 doesn't support it
+  List<dynamic> get _values => _keys.map((key) => this[key]).toList();
```

Comment:
I'd like to see us eventually polyfill `Object.values` on the JS side so that this can be potentially more optimized in browsers that support it, but this is totally fine for now